docs/developer_guide/concepts.md

```diff
@@ -36,8 +36,10 @@ where:
 
 * `environment`: the name of the environment (development, feature, staging,
   preprod or production)
-* `customer`: the one-word name of the customer site (fun, campus, corporate,
-  ademe, etc.)
+* `customer`: the one-word name of the customer site (fun, campus, ademe, 
+etc.). Max length for the customer name is 6. Because mysql username has a 
+length constraint we can not have a customer name higher than 6 characters
+long.
```

Comment:
Suggestion:

```
* `customer`: the one-word name of the customer site (_e.g._ `ucsd`, `hvrd`,
  `uca`). We enforce this name to at most 6 characters, because it is used in a
  strict database user & name pattern that must also include the environment and
  targeted application name within 16 characters (yes MySQL we are looking at
  you 😉).
```